### ISSUE TITLE
Focus is not leaving the main content in multiple dropdown menus

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9017]
## Observed behavior
In most (if not, all) parts of Kolibri outside of the navbar and appbar, if a user is navigating with a keyboard, when they tab into anything with a dropdown menu, the focus is not gained by the menu component, but instead, stays in the main content, and the user cannot use any keys to navigate within the menu.

Some examples include:

|Facility > Users|
|--|
|![2022-01-20 14 17 06](https://user-images.githubusercontent.com/13563002/150431100-7e12f642-45e7-4de7-8262-8fec4280a472.gif)|
|Device > Manage Device Channels|
|![2022-01-20 14 12 44](https://user-images.githubusercontent.com/13563002/150430597-99a78a62-278d-4cb1-af43-77df62719025.gif)|

## Errors and logs
No errors in console

## Expected behavior
User should be able to navigate through a page and when a user enters into a dropdown menu, they should be able to access the contents of the dropdown menu immediately (focus on keyboard should go straight into the menu and should be navigable using the arrow keys).

## User-facing consequences
See expected behavior. We want to fix this for accessibility.

## Steps to reproduce
See screenshots

## Context
Kolibri 0.15
Chrome
Mac OS

**Comment by marcellamaki** [comment]
Closed with https://github.com/learningequality/kolibri-design-system/pull/185
